examples/cluster/azure/variables.tf

```diff
@@ -45,7 +45,7 @@ variable "disk" {
   description = "All nodes' disk configuration"
 
   type = object({
-    size          = optional(number)
+    storage       = optional(number)
```

Comment:
with the renaming of this, how do we ensure backwards compatibility?

IMO it is ok, but we need to be very clear that this gets mentioned properly in our release notes and we do a major version update...
FYI @rajagp @a-atri

Reply:
I will rename to `size` for backwards compatibility